docs/sources/administration/provisioning/index.md

```diff
@@ -419,6 +419,10 @@ providers:
 To provision dashboards to the root level, store them in the root of your `path`.
 {{% /admonition %}}
 
+{{% admonition type="note" %}}
+This feature does not currently allow you to create nested folder structures e.g. folders within folders.
+{{% /admonition %}}
```

Comment:
- The percents aren't actually needed (they are a hangover from a previous understanding of how shortcodes work).
- Avoid e.g./i.e. https://developers.google.com/style/abbreviations#dont-use
  In this case, you actually mean i.e. (that is) not .e.g. (for example). This is one of the reasons the use of the Latin is discouraged because it's easy to confuse the two.

{{< admonition type="note" >}}
This feature doesn't currently allow you to create nested folder structures where you have folders within folders.
{{< /admonition >}}